source/includes/code-examples/DeleteOne.cs

```diff
@@ -0,0 +1,77 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
+
+namespace CSharpExamples.UsageExamples;
+
+public class DeleteOne
+{
+    private static IMongoCollection<Restaurant> _restaurantsCollection;
+    private static string _mongoConnectionString = "<Your MongoDB URI>";
+
+    public static void Main(string[] args)
+    {
+        Setup();
+
+        var doc = _restaurantsCollection.Find(Builders<Restaurant>.Filter
```

Comment:
I doubt such formatting is correct, at the very least it should be:

     var doc = _restaurantsCollection.Find(
          Builders<Restaurant>.Filter.Eq(r => r.Name, "Ready Penny Inn")).First();

to not split logically the same line about definition filter and his configuration on 2 different line

Reply:
but do we want to define more straight rules (not necessary the same as we use in the driver)? cc: @JamesKovacs

Reply:
For readability, I would suggest moving the builder expression into a `filter` variable the same way as we did it for `DeleteOne` below.